graphene_django/__init__.py

```diff
@@ -1,7 +1,7 @@
 from .fields import DjangoConnectionField, DjangoListField
 from .types import DjangoObjectType
 
-__version__ = "2.15.0"
+__version__ = "2.16.0"
```

Comment:
It's better to update when we decided to release.
```suggestion
__version__ = "2.15.0"
```